src/main/java/pl/mimuw/zpp/quantumai/backendui/controller/GraphController.java

```diff
@@ -21,16 +27,39 @@ public ResponseEntity<String> createGraph(@RequestBody EuclideanGraph graph) {
         );
     }
 
+    @PostMapping("/map")
+    public ResponseEntity<String> createMapGraph(@RequestBody MapGraphCreationRequestDto request) {
+        return ResponseEntity.ok(
+                mapGraphService.createGraph(request)
+        );
+    }
+
+    @Deprecated
     @GetMapping("/all")
     public ResponseEntity<List<EuclideanGraph>> getGraphs() {
         return ResponseEntity.ok(
-                euclideanGraphService.getGraphs()
+                euclideanGraphService.getAllGraphs()
+        );
+    }
+
+    @GetMapping("/all/new")
+    public ResponseEntity<List<GraphDto>> getGraphsOld() {
+        return ResponseEntity.ok(
+                graphService.getAllGraphs()
         );
     }
 
+    @Deprecated
     @GetMapping("/")
-    public ResponseEntity<EuclideanGraph> getGraph(@RequestParam String graphId) {
-        return euclideanGraphService.getGraph(graphId)
+    public ResponseEntity<EuclideanGraph> getGraphOld(@RequestParam String graphId) {
+        return euclideanGraphService.getGraphById(graphId)
+                .map(ResponseEntity::ok)
+                .orElse(ResponseEntity.notFound().build());
+    }
+
+    @GetMapping("/new")
+    public ResponseEntity<GraphDto> getGraph(@RequestParam String graphId) {
+        return graphService.getGraphById(graphId)
                 .map(ResponseEntity::ok)
                 .orElse(ResponseEntity.notFound().build());
     }
```

Comment:
@zemdam nowe endpointy będą prawie jak stare tylko owrapowane:
jak wcześniej było
```
{
graf
}
```
to teraz powinno być
```
{
graphType: EUCLIDEAN,
graph: { graf }
}
```